docs(README): update grammar / wording (#12)
catppuccin/conky: README.md

## Usage

> 1. Copy the contents of your prefered style (ex. `mocha.conf`) into `$HOME/.config/conky/<stylename>.conf`.

Reviewer: ```suggestion
1. Copy the contents of your preferred style (ex. `mocha.conf`) into `$HOME/.config/conky/<flavour>.conf`.
```

> 2. Run using `conky -c $HOME/.config/conky/<stylename>.conf`

Reviewer: ```suggestion
2. Run using `conky -c $HOME/.config/conky/<flavour>.conf`
```

> 1. Copy the contents of your preferred style (ex. `mocha.conf`) into `$HOME/.config/conky/<flavour>.conf`.
> 2. Run using `conky -c $HOME/.config/conky/<flavour>.conf`

Reviewer 2: ```suggestion
1. Copy the contents of your preferred style (ex. `mocha.conf`) into `$HOME/.config/conky/<flavor>.conf`.
2. Run using `conky -c $HOME/.config/conky/<flavor>.conf`
```

Author: This is a matter of British vs American spelling and is (i think) outside of the scope of this PR, as both are correct.

Reviewer 2: We typically use the American spelling for consistency across Catppuccin repos, but yeah it was just a suggestion not a requirement.
